addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -3509,3 +3509,22 @@ def test_date_reversal_exchange_move(self):
 
                 # Date of the reversal of the exchange move should be the last day of the month/year of the payment depending on the sequence format
                 self.assertEqual(exchange_move_reversal.date, fields.Date.to_date(expected_date))
+
+    def test_invoice_mass_posting(self):
+        currency = self.company_data['currency']
+        currency_previous_rounding = currency.rounding
+        currency.rounding = 0.0001
+        invoice1 = self.init_invoice(move_type='out_invoice', partner=self.partner_a, invoice_date='2016-01-20', products=self.product_a)
+        invoice1.invoice_line_ids.price_unit = 12.36
+        invoice2 = self.init_invoice(move_type='out_invoice', partner=self.partner_b, invoice_date='2016-01-20', products=self.product_a)
+
+        moves = invoice2 | invoice1
+        moves._post(soft=False)
```

Comment:
Instead of calling a private method, use something the users have access to: `action_post` (black box testing vs white box testing)